app/helpers/insured/families_helper.rb

```diff
@@ -218,7 +218,8 @@ def build_link_for_sep_type(sep, link_title=nil)
       # Take straight to the Plan Shopping - Add Members Flow. No date choices.
       # Use turbolinks: false, to avoid calling controller action twice.
       # TODO: Refactor Shop For Planss as a translation at some point
-      link_to link_title.presence || 'Shop for Plans', insured_family_members_path(sep_id: sep.id, qle_id: qle.id), class: 'btn btn-default', data: {turbolinks: false}
+      link_path = family.present? ? insured_family_members_path(sep_id: sep.id, qle_id: qle.id, family_id: family.id) : insured_family_members_path(sep_id: sep.id, qle_id: qle.id)
```

Comment:
Instead of passing family object to this method, can you just send the family id?

Reply:
since family id is passed in, we can directly use family_id, instead of family.id